pkg/aspect/quotasManager_test.go

```diff
@@ -193,7 +194,8 @@ func TestQuotaWrapper_Execute(t *testing.T) {
 		{goodMd, 1, nil, false, errEval, make(map[string]o), "expected"},
 		{goodMd, 1, nil, false, labelErrEval, make(map[string]o), "expected"},
 		{goodMd, 1, nil, false, goodEval, map[string]o{"request_count": {1, []string{"source", "target"}}}, ""},
-		{goodMd, 0, fmt.Errorf("alloc-forced-error"), false, goodEval, map[string]o{"request_count": {1, []string{"source", "target"}}}, "alloc-forced-error"},
+		{goodMd, 0, errors.New("alloc-forced-error"), false, goodEval, map[string]o{"request_count": {1, []string{"source", "target"}}},
```

Comment:
nit: this looks weird with the new line break.  should we move the `map[string]o{"request_count": {1, []string{"source", "target"}}}` up (as it is used in another test case, too)?